### PR TITLE
Fix for issue #132

### LINKED ISSUE #132: Test fails at creating appointments
For example: https://travis-ci.org/simplyGits/MagisterJS/builds/528474052

Can anybody with a working account figure this out?

### PR DESCRIPTION
**Pull request by Argetan**
This time I'm pretty sure I really fixed #132.

### CODE REVIEW

File: src/magister.js [hunk 1]
```diff
@@ -274,21 +274,22 @@ class Magister {
...
 			Lokatie: _.trim(options.location),
-			Inhoud: (function () {
-				const content = _.trim(options.content)
-				return content.length > 0 ? _.escape(content) : null
-			})(),
+			Inhoud: content,
 			Type: options.type || 1,
 			DuurtHeleDag: options.fullDay || false,
 
+			InfoType: content === null ? 0 : 6,
```
Comment (lieuwex) [inline src/magister.js]: Interesting!

File: test/test.js [hunk 2]
```diff
@@ -240,23 +242,60 @@ describe('Magister', function() {
...
+				description: description,
+			}).then(response => {
+				expect(response.statusText).to.equal('Created')
+
+				return m.appointments(yesterday, tomorrow).then(appointments => {
+					const appointment = appointments.find(appointment => appointment.description === description)
+
+					return appointment.remove().then(response => {
+						// TODO: review: is this the most accurate way to test this?
+						expect(response.statusText).to.equal('No Content')
```
Comment (lieuwex) [inline test/test.js]: Maybe this is also checkable with the status code?

File: test/test.js [hunk 3]
```diff
@@ -240,23 +242,60 @@ describe('Magister', function() {
...
-					expect(appointment).to.exist
-					return appointment.remove()
+				start: start,
+				end: end,
+				description: description,
+			}).then(response => {
+				expect(response.statusText).to.equal('Created')
+
+				return m.appointments(yesterday, tomorrow).then(appointments => {
+					const appointment = appointments.find(appointment => appointment.description === description)
```
Comment (lieuwex) [inline test/test.js]:
```suggestion
					const appointment = appointments.find(a => a.description === description)
```

File: test/test.js [hunk 4]
```diff
@@ -240,23 +242,60 @@ describe('Magister', function() {
...
-			}).then(() => {
-				return m.appointments(today, nextWeek).then(r => {
-					const appointment = r.find(a => a.description === description)
-					expect(appointment).to.exist
-					return appointment.remove()
+				start: start,
+				end: end,
+				description: description,
+			}).then(response => {
+				expect(response.statusText).to.equal('Created')
```
Comment (lieuwex) [inline test/test.js]: Maybe this is checkable with the status code?

File: test/test.js (hunk 2 above)
Comment (Argetan) [reply test/test.js]: Well the `statusText` is just a text version of the `statusCode`, I think doing it this way is clearer (you instantly see that the status is `No Content` whereas `204` is something I'd have to look up to see what it means again) and should be as accurate. What inspired me to use this is the [`Django REST framework`](https://www.django-rest-framework.org/api-guide/status-codes/).

Comment (lieuwex) [reply test/test.js]: Fair enough!

**Pull request merged**